jekyll/_cci2/api_v2_introduction.md

```diff
@@ -0,0 +1,226 @@
+
+#### What Does This Change for Me? How Do I Turn On Pipelines?
+
+Whatever you are running today should continue working without any intervention. CircleCI has some features (such as 2.1 configuration and orbs) that require you to enable pipelines. In some rare cases, some users may need to make tweaks to their configuration when enabling pipelines.
+
+Many users are already using an early version of pipelines, even before CircleCI exposed them through the CircleCI API and web UI. If you have already used the latest features in CircleCI version 2.1 configuration and/or orbs, then you already have pipelines enabled for your project (all projects added after September 1, 2018 already have pipelines enabled by default). If, however, your project does not yet have pipelines enabled, go to **Advanced Settings** in your project's **Settings** page. If you do not see the **Enable Pipelines** option on your project's settings page, this means your project already has pipelines enabled.
+
+#### Is a Pipeline the Same Thing as a Build?
+
+In previous CircleCI releases, the term "build" was used to describe a pipeline. Unfortunately, this terminology proved problematic because there was an incongruity between how "build" and "job" were defined in the API and UI. For example, what was referred to as a "build" in past releases is now referred to as a "job" in the current release. To address this ambiguity, CircleCI updated the UI to reflect this change in nomenclature, but there are still instances of "build" in our API.
+
+More importantly, CircleCI found that more teams are using CircleCI for a much broader array of automated processes than the word "build" implies. “Pipelines” more accurately reflects the broad applications of our platform, therefore, "pipelines" are more appropriate for the set of stages CircleCI performs in processing your automation instructions against changes in your code. 
+
+#### Will I Have to Use Pipelines?
+
+You are not required to use pipelines, but note that pipelines will eventually be enabled for all projects. This will allow CircleCI to provide more powerful configuration and automation, as well as better optimization of your resources and more reliable performance. The overarching goal of this approach is for the transition to pipelines to be seamless and have no disruptions on any of your existing workflows.
+
+### Pipeline Parameters
```

Comment:
Recommend this gets documented separately -- formally speaking it's not part of the "API", though when in the API we show passing them in a trigger we should link to the config documentation about pipeline parameters in config (and vice versa).

Reply:
Removed from page. Will add to separate page.